HomeWork5.py

```diff
@@ -0,0 +1,183 @@
+#HomeWork4.1
+firstData = 0.0
+secondData = 0.0
+def plus( first=0, second=0):
+    print("additional = ", first+second)
```

Comment:
Can be simplify
Function:
```suggestion
    return first+second
```

Reply:
These changes can be applied to all functions